### PR TITLE
Add private key warning screen to help stop scammers

### PR DESCRIPTION
**Pull request by Perronef5**
## CHANGES 📲
- Add Private Key warning Screen to both RevealWords & RevealPrivateKey Screens

## DEMO 📸
Simulator Screen Recording - iPhone 15 Pro - 2024-01-15 at 22 57 45

## Why?
Scammers have been asking users to share with them there private keys or secret phrase. Non crypto savvy users don't understand the risk of sharing this information. This modal is to help educate users on the risk of sharing this information.

### CODE REVIEW

**Comment by mawdegroot** [comment]
nitpicking: Helium misses the capital H. Can we add a timer (e.g. 5 seconds) before you can press "Proceed with caution" too?

**Comment by abhay** [comment]
> nitpicking: Helium misses the capital H. Can we add a timer (e.g. 5 seconds) before you can press "Proceed with caution" too?

I like the timer idea a lot.

File: src/locales/en.ts [hunk 1]
```diff
@@ -461,6 +461,11 @@ export default {
       'Please be advised that the subDAO treasury has only been active for a few days since the Solana migration. Swapping subtokens to HNT may be affected by a limited HNT supply in the subDAO treasury and may not reflect up-to-date prices. Please proceed with caution.',
     addRecipient: 'Add Recipient',
   },
+  secretKeyWarningScreen: {
+    title: 'Warning! Nobody should ask for your secret phrase or private key.',
+    body: 'Would you give the username and password to your bank account? The answer is no. Nobody should have your secret phrase. Not even helium employees.',
```
[the remark this answers is not in the document]
Comment (Perronef5) [reply src/locales/en.ts]: Done

**Comment by Ryan-Goldstein** [comment]
May be a bit late, but please add copy for entering words on a website too. I'd say 98% of the time, scammers direct people to enter their words on a website.

Numerous times, I've seen conversations where the victim said something along the lines of "I'm not supposed to share those words with anyone", and the scammer replies along the lines of, "Don't worry, no one will see those words, it's an encrypted website to rectify the API wallet backend."

So, the emphasis should ideally be on not entering those words on any website, rather than not giving them to someone.

**Pull request merged**